### ISSUE TITLE
Ender IO Item Conduit Will Not Extract From A Chest

### ISSUE THREAD

**Issue by SmugDragon** [CrazyPants/EnderIO#2066]
I am currently playing FTB Infinity 1.2.0, the latest version I can find, and discovered the error while building a tree farm. Items flow from machines, in my case the Harvester machine, w/o issue into a chest. Yet when I try to extract Oak Logs from that chest into my furnace, nothing happens. I doubled then triple checked the settings and nothing that I've tried worked. I thought perhaps it was my world, so I created a new test environment, went in creative and connected 2 chests with a single Ender IO item conduit, again making sure one side was set to extract and the other to input...nothing happened. I then tried adding filters, but this failed as well. I have reported this issue on the FTB support site, but since all the other IO conduits work (fluid runs like a charm) I thought I would post it here as well. Any help would be wonderful!
Thanks,
Smug

**Comment by HenryLoenwind** [comment]
Hi. This is a little bit weird because that is such a basic functionality---it cannot be broken for everyone or we would have noticed. So could you please provide screenshots of the conduit connections and your test setup?

BTW, you *did* set them do extract "without Redstone signal" or "always"?

**Comment by SmugDragon** [comment]
Hey Henry,
Well I don't know what to say besides I must be losing my mind. I've had this issue for 2 days now and I swear I checked the extraction w/o redstone signal settings not once, not twice, but three separate times. I even rebuilt it in my test world...yet when I logged in today, they were all set incorrectly. As soon as I reset them, everything worked and it's still working. I apologize for bothering you with what I can only call a "derp" moment. >.< But I thank you none the less for your help.
Cheers!
Smug
